### PR TITLE
[CR] Wool take two

### PR DESCRIPTION
**Pull request by PropaneSoup**
This adds everything for adding wool as a custom modification, including an allergy check. The one thing that may or may not be for this pull request is discussing how to handle encumbrance. I think it would be best to increase the granularity of encumbrance by 10, so that adding a fur lining could give .7 encumbrance, while leather padding could be .3. I think this is a much better solution than giving a crippling +1 to clothing every time you modify something.

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
>discussing how to handle encumbrance

Increased granularity to the encumbrance system has been discussed before, and would be really nice to have, but that's outside the scope of this PR.

File: src/item.cpp [hunk 1]
```diff
@@ -2336,13 +2345,15 @@ int item::get_warmth() const
...
 
     if (item::item_tags.count("furred") > 0){
-        warmed = 2 * volume() * (float(get_coverage()) / 100);
-        return result + warmed;
-    } else {
-        return result;
+        fur_lined = 2 * volume() * (float(get_coverage()) / 100);
+    }
+    if (item::item_tags.count("wooled") > 0){
+        wool_lined = 1.5 * volume() * (float(get_coverage()) / 100);
```
Comment (KA101) [inline src/item.cpp]: Ahem.  Volume doesn't make much sense in this context.  No need to keep perpetuating the last time I communicated poorly.  :-(

**Comment by KA101** [comment]
I'm not opposed to the granularity increase but it's not happening until after the stable lands.  Apart from that and the volume problem, looks OK but I'm not sure re freeze.

**Comment by PropaneSoup** [comment]
It's alright I figured this wasn't getting in because of the freeze.

**Comment by kevingranade** [comment]
We're theoretically at T-3 days, nothing but bugfixes should be going in
right now.

**Comment by KA101** [comment]
OK, you hadn't thrown the FREEZE tags in that forum thread last I checked.  I'll take care of that now if they're still showing no-freeze.

**Comment by PropaneSoup** [comment]
Wait, now I'm confused. Are we talking about the feature freeze, or is there a freeze item tag that was discussed? In any case, thanks for taking care of anything I forgot.

**Comment by KA101** [comment]
Feature, and now string/content freeze.  Apologies for the confusion.

**Comment by KA101** [comment]
This one passed.  Sorry it didn't merge, probably my missing it (needed four test-branches, grr).

**Pull request merged**